pages/docs/linkfree-profile-tips.mdx

```diff
@@ -0,0 +1,21 @@
+import DocsLayout from "@components/layouts/DocsLayout.js"; import Link from "@components/Link";
+
+# Linkfree Profile Tips 
+
+Trying to make your Linkfree profile stand out, but have no idea how? Check out this these tips below:
+
+## Tip #1: Make it easier for people to follow your socials
+- Do you have a YouTube channel and/or Twitter profile?  Add the link for the subscribe flag, so when they visit your channel you can click right on the subscription or follow button. Check out [this article](https://offeo.com/learn/how-to-create-a-subscribe-link-for-youtube) and [this example](https://www.indiehackers.com/post/twitter-tip-link-to-the-intent-url-instead-of-your-username-7e34f7b077) to create your Twitter follow intent link.
+- If you use more than one blog, consider putting them into the different sections. It'll help people know which one to refer to in order to follow your journey. Check out [Christine's profile for inspiration](https://linkfree.io/CBID2).
+## Tip #2: Share your achievements and future plans
+- Whether it's reaching 1K followers on Twitter, winning a contest, or creating a project that helped your employer achieve company goals, share your achievements in the **Milestones** tab! :) Check out [Krupali's profile](https://linkfree.io/krupalitrivedi) for ideas. 
+- Want to do better in school?  Win that marathon? Cook a full meal? Or get that dream job? Share those too! You can do that by creating a future goals section underneath the ones you have achieved. 
+Check out [Eddie's profile for inspiration](https://linkfree.io/eddiejaoude). 
+## Tip #3: Share your recommendations and Events
+- Trying to increase your prospects in your job search? Want to grow your business? Then, consider having the people in your network add their reviews in the **Testimonials** section!  Check out [Francesco's profile for inspiration](https://linkfree.io/FrancescoXX). 
+- Want to do in-person meetups with your online colleagues or your social media followers? Then consider adding them in the **Events** section! Check out [Amanda's profile](https://linkfree.io/amandamartin-dev) for inspiration! 
+
+
+
+
+  
```

Comment:
These extra lines are not needed. Also I think all docs files have some code at the end to make them work

Reply:
Thanks @eddiejaoude! :) I'll have to check what those lines are.